### ISSUE TITLE
rollout_options does not allow empty value, when performing plan a change is detected

### ISSUE THREAD

**Issue by nbrys** [observIQ/terraform-provider-bindplane#106]
**Describe the bug**
Having the following configuration block in terraform without rollout_option:

```
resource "bindplane_configuration" "configuration" {
  for_each   = var.configurations
  name       = each.key
  platform   = each.value.platform
  labels     = each.value.labels
  rollout    = each.value.rollout
  extensions = each.value.extensions
  dynamic "source" {
    for_each = each.value.sources != null ? each.value.sources : []
    content {
      name       = source.value.name
      processors = source.value.processors
    }
  }

  dynamic "destination" {
    for_each = each.value.destinations != null ? each.value.destinations : []
    content {
      name       = destination.value.name
      processors = destination.value.processors
    }
  }
}
```

A plan of this produces:
```
# bindplane_configuration.configuration["test"] will be updated in-place
  ~ resource "bindplane_configuration" "configuration" {
        id           = "idhere"
        name         = "cph-proxy"
        # (4 unchanged attributes hidden)

      - rollout_options {
        }

        # (4 unchanged blocks hidden)
    }
```
**To Reproduce**
Create a configuration with rollout_option block and perform a terraform plan.

**Expected behavior**
No change on the configuration

**Screenshots**
/
**Environment:**

Terraform 1.6.6 and provider version 1.4.0

**Comment by jsirianni** [comment]
@nbrys thank you for reporting. We will look into replicating this issue.

**Comment by jsirianni** [comment]
I have re-produced this, working on a solution :+1:

**Comment by jsirianni** [comment]
@nbrys this should be resolved in v1.4.3 https://github.com/observIQ/terraform-provider-bindplane/releases/tag/v1.4.3
